### PR TITLE
i did it again

### PR DESCRIPTION
**Pull request by SOMEWANTTOSMILE**

### CODE REVIEW

**Comment by Misnad** [comment]
What have you done again? Please.

File: Animals_class/Animal_class.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class Moving:
+    def move(self):
+        raise NotImplementedError(
+            'Define voice in %s.' % self.__class__.__name__)
+
+
+class Animal(Moving):
+    def voice(self):
+        super().move()
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: Nope.. voice and move it is different methods.. why are you calling move from voice?

File: Animals_class/Animal_class.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+class Transport(Moving):
+    def launch(self):
+        super().move()
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: also here..

File: Animals_class/Animal_class.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    def voice(self):
+        print("tiger growls")
+
+    def move(self):
+        print("Tiger is running")
+
+
+class Car(Transport):
+    def __init__(self, status="non launch"):
+        self.status = status
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: Where we creating car - we cant define its status.. It looks like I can launch my car without `launch` method
It is bad.. 🤨

File: Animals_class/Animal_class.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+class Tiger(Animal):
+    def voice(self):
+        print("tiger growls")
+
+    def move(self):
+        print("Tiger is running")
+
+
+class Car(Transport):
+    def __init__(self, status="non launch"):
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: it is not recomending define strings variables like that..
Better to use None by default and define it below using `if` operator

Comment (nicilytop-tms) [reply Animals_class/Animal_class.py]: But see comment below

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]
check coments please and do another commit..

File: Animals_class/Animal_class.py [hunk 5]
```diff
@@ -6,12 +6,14 @@ def move(self):
 
 class Animal(Moving):
     def voice(self):
-        super().move()
+        raise NotImplementedError(
+            'Define voice in %s.' % self.__class__.__name__)
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: lets move last bracket below

File: Animals_class/Animal_class.py [hunk 6]
```diff
@@ -6,12 +6,14 @@ def move(self):
...
 
 
 class Transport(Moving):
     def launch(self):
-        super().move()
+        raise NotImplementedError(
+            'Define voice in %s.' % self.__class__.__name__)
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: and here

File: Animals_class/Animal_class.py [hunk 7]
```diff
@@ -41,9 +43,9 @@ def move(self):
             print("Car is rides")
 
     def launch(self):
-        if self.status == 'non launch':
+        if self.status is None:
             self.status = 'Launch'
             print("launch")
-        elif self.status == 'Launch':
+        elif self.status is not None:
```
Comment (nicilytop-tms) [inline Animals_class/Animal_class.py]: we don't need this check.. it looks like if I want mute my car and status is `non launch` I will set status `non launch` contrary to that this status already setted

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]
do it again please

**Review by nicilytop-tms — APPROVED** [review]
ok ok

**Pull request merged**